argnorm/drug_categorization.py

```diff
@@ -7,27 +7,36 @@
 
 def confers_resistance_to(aro_num: str) -> List[str]:
     '''
-    Description: Returns a list of the drugs/antibiotics to which a gene confers resistance to.
+    Description: Returns a list of the drugs/antibiotics to which a gene and its parents confers resistance to.
```

Comment:
I think this is technically incorrect. It returns the list of drugs to which a gene confers resistance to. The fact that ARO encodes that information in the parents is a bit of an internal detail